src/otk/command.py

```diff
@@ -49,6 +52,8 @@ def _process(arguments: argparse.Namespace, dry_run: bool) -> int:
     else:
         paths = extra + [pathlib.Path(arguments.input)]
 
+    log.info(f"Compiling {paths}")
```

Comment:
Fwiw, AIUI info is not shown by default (which may or may not what we want here) - having a simple smoke test would make the intend clear :)